addons/hr_gamification/static/src/messaging_menu_patch.js

```diff
@@ -10,11 +10,11 @@ patch(MessagingMenu.prototype, {
         this.orm = useService("orm");
     },
 
-    onClickThread(isMarkAsRead, thread) {
+    onClickThread(isMarkAsRead, thread, message) {
         if (!isMarkAsRead && thread.model === "gamification.badge.user") {
             this.openEmployeeView(thread);
         } else {
-            super.onClickThread(isMarkAsRead, thread);
+            super.onClickThread(isMarkAsRead, thread, message);
```

Comment:
I would `...arguments` to ensure the patch doesn't compromise change of params there.